spec/services/search/postgres/comment_spec.rb

```diff
@@ -0,0 +1,118 @@
+require "rails_helper"
+
+RSpec.describe Search::Postgres::Comment, type: :service do
+  let(:comment) { create(:comment) }
+
+  describe "::search_documents" do
+    context "when filtering Commentables" do
+      it "does not include comments from Articles that are unpublished", :aggregate_failures do
```

Comment:
Should we just tag _all_ specs with `:aggregate_failures`? I see it a lot in out codebase and we can do it globally if that would be easier than remembering to do it everywhere.

Reply:
That's a great point/question! Maybe @djuber has thoughts on this?